### ISSUE TITLE
Plot templates

### ISSUE THREAD

**Issue by nicolasaunai** [SciQLop/SciQLop#9]
When looking at data it is often that we browse the same quantities in the same plot layout. Opening sciqlop, it is a bit annoying to have to re-make that from the data tree all over again.

It would be nice to have the possibility to save a plot panel as a template. Templates would store the variables used, the plot properties (layout, order, which variable is on which plot etc.), not necessarily the time interval. The idea is to be able to save templates and browse them, have favorites...

**Comment by jeandet** [comment]
Depends on #10

**Comment by jeandet** [comment]
Can we consider that Workspaces+Jupyter notebooks cover this?

**Comment by jeandet** [comment]
Done, from the welcome panel:
<img width="1180" height="325" alt="Image" src="https://github.com/user-attachments/assets/02a5c577-bbbe-4a9a-b012-af4073430218" />

Any panel can be converted into a template as long as there are no virtual products (they will be ignored).
<img width="1174" height="577" alt="Image" src="https://github.com/user-attachments/assets/34545c10-8a69-48cd-bc45-6d33c394ff72" />
